### ISSUE TITLE
Add details about the project and Contributing Guidelines in the readme as it will be easier for contributors.

### ISSUE THREAD

**Issue by AnirudhPanda** [Free-Pustakalya/freepustakalya.website#2]

**Comment by yashsehgal** [comment]
Thanks for raising this issue @AnirudhPanda 👍🏽 
You can help us with this issue. Maybe we can start a discussion about the same.

**Comment by AnirudhPanda** [comment]
> Thanks for raising this issue @AnirudhPanda 👍🏽
> You can help us with this issue. Maybe we can start a discussion about the same.

Yeah sure! If you give me all the details about this project. Would love to know what is your objective!?
